e2e/.env.e2e

```
@@ -80,17 +80,24 @@ DB_CERTIFICATE_NAME=prompt
 # Empty = same-origin: the component's API calls go to the browser origin
 # (client-core) and are proxied to the phase server by e2e/nginx/client-core.conf.
 SELF_TEAM_ALLOCATION_HOST=
 ASSESSMENT_HOST=
 INTERVIEW_HOST=
 CERTIFICATE_HOST=
+TEAM_ALLOCATION_HOST=
 # The remaining modules are not part of the e2e stack (yet).
 INTRO_COURSE_HOST=http://localhost:8082
 DEVOPS_CHALLENGE_HOST=http://localhost:9010
-TEAM_ALLOCATION_HOST=http://localhost:8083
 EXAMPLE_HOST=http://localhost:8086
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXAMPLE_HOST key should go before the INTERVIEW_HOST key

(UnorderedKey)